cpp/distance_calculator/distance_calculator.cpp

```diff
@@ -0,0 +1,136 @@
+#include <mgp.hpp>
+#include <math.h>
+# define M_PI 3.14159265358979323846  /* pi */
```

Comment:
Isn’t π already defined by including math.h?

Reply:
yes, you are right